pages/price-feeds/best-practices.mdx

```diff
@@ -20,6 +20,9 @@ The confidence interval is `1500 * 10^(-5) = $0.015`, and the price is `12276250
 Sometimes, Pyth will not be able to provide a current price for a product.
 This situation can happen for various reasons.
 For example, US equity markets only trade during certain hours, and outside those hours, it's not clear what an equity's price is.
+Pyth price feeds follow the [traditional market hours](https://www.cmegroup.com/trading-hours.html) for each asset class. \
```

Comment:
minor thing: I don't think you want to link to the CME here, because this is duplicating the info about the reference source for the market hours -- just send them to our market hours page and that links to CME.